### PR TITLE
Use bazel-like space separated target specification

### PR DESCRIPTION
**Pull request by djmarcin**
I gave up on the ideal of &str because I was wrestling the type system getting increasingly absurd and it just didn't seem worth it for this.

### CODE REVIEW

File: tools/rust_analyzer/lib.rs [hunk 1]
```diff
@@ -53,7 +53,7 @@ pub fn write_rust_project(
         rules_rust_name.as_ref(),
     )?;
 
-    let workspace_name = match rules_rust_name.as_ref().trim_start_matches("@") {
+    let workspace_name = match rules_rust_name.as_ref().trim_start_matches('@') {
```
Comment (djmarcin) [inline tools/rust_analyzer/lib.rs:56]: Fixes clippy from the last PR

File: tools/rust_analyzer/main.rs [hunk 2]
```diff
@@ -112,6 +115,6 @@ struct Config {
     bazel: PathBuf,
 
     /// Comma-separated list of target patterns
```
Comment (abrisco) [inline tools/rust_analyzer/main.rs]: This comment is no longer correct, right?

**Review by abrisco — CHANGES_REQUESTED** [review]
Sounds reasonable to me!

File: tools/rust_analyzer/main.rs (hunk 2 above)
Comment (djmarcin) [reply tools/rust_analyzer/main.rs]: fixed

**Pull request merged**